packages/vm/src/cache.rs

```diff
@@ -125,7 +126,7 @@ where
         }
 
         // Get module from file system cache
-        if let Some(module) = self.fs_cache.load(checksum)? {
+        if let Some(module) = self.fs_cache.load(checksum, options.memory_limit)? {
```

Comment:
This take an optional config memory limit (very nice), why not config above?

Reply:
This limit is not optional but required for every instanciation.
You can have a single cache and get instances with different memory limits out of it. Not that we need it but I ran into design trouble when I had a fixed limit in the cache.